step_definitions/cucumber_mappings.rb

```diff
@@ -50,7 +50,7 @@ def step_file(pattern)
   end
 
   def pattern_exists?(pattern)
-    File.exist?(File.join(current_dir, step_file(pattern)))
+    File.exist?(File.join(expand_path("."), step_file(pattern)))
```

Comment:
After merging and rebasing a few PRs I lost track of what was where. Everyone makes mistakes ;-)